src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/AssessmentApiDataService.kt

```diff
@@ -16,15 +18,17 @@ class AssessmentApiDataService(
 ) {
 
   fun isAssessmentRecent(crn: String): Boolean {
-    return assessmentApiClient.getLatestAssessment(crn).let {
+    return getLatestCompletedAssessment(crn).let {
       val cutOff = LocalDate.now(clock).minusWeeks(55).minusDays(1)
-      it.completed.toLocalDate().isAfter(cutOff)
+      // Will be not null because it is completed
+      it.completed!!.toLocalDate().isAfter(cutOff)
```

Comment:
if it's never null here (although it looks like it can be when retrieved from the API), it might be an idea to introduce another type with a non-nullable when processing these assessments. Seems a  shame to have to do this otherwise

This assessments API gives back some very horrible data